hinted-compilation/get-hints.py

```diff
@@ -525,7 +534,7 @@ def _ourimport(
 
 try:
     proc = subprocess.Popen(new_argv)
-    proc.wait(timeout=5 * 60)
```

Comment:
So this is only about making it configurable. I guess, adding a trace about the timeout value, and when it happens, an information would be nice, or will `proc.wait` raise?

Reply:
Proc.wait would raise the same exception as before, which would say times out after x seconds. I can add a line to print the timeout value though, that'd be nice especially since the default is 5 minutes, not infinity.